metrics/ci_integrations/test/client/buildkite/buildkite_client_test.dart

```diff
@@ -113,6 +114,22 @@ void main() {
       expect(client.authorization, equals(authorization));
     });
 
+    test(
+      ".headers contain HttpConstants.defaultHeaders as a default value",
```

Comment:
```suggestion
      ".headers contain HttpConstants.defaultHeaders if headers parameter is not specified",
```
Would it be a bit more clear?